test/bondenv/optimize.jl

```diff
@@ -0,0 +1,45 @@
+using Random
+using Printf
+using Test
+using TensorKit
+using PEPSKit
+using LinearAlgebra
+using KrylovKit
+
+Random.seed!(10)
+maxiter = 500
+check_interval = 20
+trscheme = truncerr(1e-10) & truncdim(8)
+Vext = ℂ[FermionParity](0 => 100, 1 => 100)
+Vint = ℂ[FermionParity](0 => 5, 1 => 5)
+Vphy = ℂ[FermionParity](0 => 1, 1 => 2)
```

Comment:
```suggestion
Vext = Vect[FermionParity](0 => 100, 1 => 100)
Vint = Vect[FermionParity](0 => 5, 1 => 5)
Vphy = Vect[FermionParity](0 => 1, 1 => 2)
```
I think the other syntax is deprecated (although there might still be references to this in the docs)